src/main/kotlin/racingcar/CarController.kt

```diff
@@ -0,0 +1,58 @@
+package racingcar
+
+import kotlin.random.Random
+
+private const val RANDOM_RANGE = 10
+
+class CarController(private val carNames: List<String>, private val playCount: Int) {
+
+    private fun makeCars(names: List<String>): List<Car> {
+        val cars = mutableListOf<Car>()
+        for (name in names) {
+            cars.add(Car(name))
+        }
+        return cars
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return names.map { Car(it) }
```